src/bsp/src/stm32/src/interloc/src/InterlocManager.cpp

```diff
@@ -41,7 +41,7 @@ void InterlocManager::startInterloc() {
     syncClocks();
 
     // Uncomment the following line to go into TWR
-    //    m_stateHandler.setState(InterlocStates::IDLE, InterlocEvent::NO_EVENT);
+    m_stateHandler.setState(InterlocStates::IDLE, InterlocEvent::NO_EVENT);
```

Comment:
Instead of alway commenting/decommenting, could we just add a check and enable the state if both DWs were initialized properly?

Reply:
will do